### ISSUE TITLE
Supported encryption standards differs from Powershell documentation

### ISSUE THREAD

**Issue by haymansfield** [Azure/azure-cli#12347]
The equivalent Powershell documentation (https://docs.microsoft.com/en-us/powershell/module/az.network/new-azipsecpolicy?view=azps-3.5.0) indicates a reduced set of supported set of standards, namely IkeEncryption and IkeIntegrity not supporting GCMAES256.

In addition https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-vpn-faq#ipsecike supports the powershell documentation. Is the documentation here incorrect?

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 74bccfe9-5e94-430b-3002-ade01e60afde
* Version Independent ID: a2784876-679c-cd4e-61ae-b793759ceb2b
* Content: [az network vpn-connection ipsec-policy](https://docs.microsoft.com/en-gb/cli/azure/network/vpn-connection/ipsec-policy?view=azure-cli-latest)
* Content Source: [src/azure-cli/azure/cli/command_modules/network/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/_help.py)
* Service: **virtual-network**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by qwordy** [comment]
@MyronFanQiu please take a look, thanks

**Comment by MyronFanQiu** [comment]
@haymansfield Yeah. I think you're right. Our documentation is coming from the swagger spec, which is also defined by service team. Looks like they don't keep the value consistent. I will change the message later or invite service team to change the swagger.

**Comment by MyronFanQiu** [comment]
in the swagger spec, IkeEncryption and IkeIntegrity support both GCMAE256 and GCMAE128.
https://github.com/Azure/azure-rest-api-specs/blob/c1ffffc63cc67b1780b4e0a41d0ce28520d9208d/specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json#L3439-L3440

https://github.com/Azure/azure-rest-api-specs/blob/c1ffffc63cc67b1780b4e0a41d0ce28520d9208d/specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json#L3455-L3456

But in the official doc https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-vpn-faq#ipsecike, they don't. Which one is correct? Please change one of them according to the real server's behavior.

**Comment by yonzhan** [comment]
Any update from Service team about it?

**Comment by ababdul** [comment]
Can we have an update on this request? Its been 3 years and no one has updated the Documentation.

**Comment by aznetsuppgithub** [comment]
+ @Rajesh ***@***.***>
